src/core/builder.ts

```diff
@@ -63,6 +63,7 @@ class DepGraphBuilder {
     this._pkgNodes[pkgId].add(nodeId);
 
     this._graph.setNode(nodeId, { pkgId, info: nodeInfo });
+    return this;
```

Comment:
can you explain why?
also - can this be tested somehow?

Reply:
The `why` is mentioned in the commit.

However, one might argue that fluent interfaces are not very common in the JavaScript ecosystem.

Reply:
IMO, this is a counter-productive thing test. 
re idiomatic JS - good ideas need opportunity to spread / evolve!